### PR TITLE
added macro {spaceless}

### PR DESCRIPTION
**Pull request by dg**
Removes extra whitespace.

### CODE REVIEW

**Comment by Majkl578** [comment]
:-1: Multiline tags are useful, especially with multiple attributes, incl. many data-*. It makes it more readable too in that case.

**Comment by dg** [comment]
Changed to macro {compress} and n:compress.

File: src/Latte/Macros/CoreMacros.php [hunk 1]
```diff
@@ -26,6 +26,7 @@
  * - {_expression} echo translation with escaping
  * - {attr ?} HTML element attributes
  * - {capture ?} ... {/capture} capture block to parameter
+ * - {compress} ... {/compress} compresses output
```
Comment (JanTvrdik) [inline src/Latte/Macros/CoreMacros.php]: „compresses output“ is confusing, what about „removes insignificant HTML whitespace“

**Comment by JanTvrdik** [comment]
What about more descriptive name `{compressTags}` or `{compressHtmlTags}`. Rarely used macros must have more descriptive names.

**Comment by dg** [comment]
It compressed whitespace ;)

**Comment by JanTvrdik** [comment]
@dg The tests are so confusing that I changed my mind twice about what I think the macro do.

**Comment by milo** [comment]
What about name `shrink`?

**Comment by dg** [comment]
It compresses spaces, so … n:vakuum? :)

**Comment by Majkl578** [comment]
Or `shrinkWhitespaces`?

`compress` is extremely ambiguous, one could think of real compression.

**Comment by dg** [comment]
What about spaceless? https://docs.djangoproject.com/en/dev/ref/templates/builtins/#spaceless

**Pull request merged**